pom.xml

```diff
@@ -266,15 +267,15 @@
         </plugin>
         <plugin>
           <artifactId>maven-jar-plugin</artifactId>
-          <version>2.4</version>
+          <version>3.2.0</version>
         </plugin>
         <plugin>
           <artifactId>maven-plugin-plugin</artifactId>
-          <version>3.5</version>
+          <version>3.5.1</version>
         </plugin>
         <plugin>
           <artifactId>maven-release-plugin</artifactId>
-          <version>2.5.3</version>
+          <version>3.0.0-M1</version>
```

Comment:
Is it also needed?

Reply:
yes, it is needed: this version of the Maven Release Plugin is the first version able to update `project.build.outputTimestamp` property value during the release

Reply:
Turns out dependable bot has updated this anyways. Can you just resolve the merge conflict to the latest version?

Reply:
done